src/services/importTracker.ts

```diff
@@ -473,7 +475,10 @@ export function findModuleReferences(program: Program, sourceFiles: readonly Sou
 
         forEachImport(referencingFile, (_importDecl, moduleSpecifier) => {
             const moduleSymbol = checker.getSymbolAtLocation(moduleSpecifier);
-            if (moduleSymbol === searchModuleSymbol) {
+            const compilerOptions = program.getCompilerOptions();
+            const jsxImport = getJSXRuntimeImport(getJSXImplicitImportBase(compilerOptions, referencingFile), compilerOptions);
+
+            if (moduleSymbol === searchModuleSymbol && moduleSpecifier.text !== jsxImport) {

```

Comment:
this will skip even explicit, non-synthetic references to the jsx implicit import, right? Is there a way to check that the import declaration is synthetic, because those are the only ones we want/need to skip.

Reply:
That's correct. Fixed.